### ISSUE TITLE
Waybar says module `cava` is unknown

### ISSUE THREAD

**Issue by kernaltrap8** [Alexays/Waybar#3491]
I really like audio visualizer programs, and I currently use GLava but then I noticed cava is supported in waybar. Decided to add the module, but waybar says the module is invalid. I have installed cava and also updated to the live version of waybar, still no luck. Below is my config.jsonc and style.css
http://0x0.st/XOXn.jsonc
http://0x0.st/XOXd.css

**Comment by kernaltrap8** [comment]
I was able to get it """working""" by actually building from git and not using the Gentoo package. The module doesnt seem to behave correctly.

https://github.com/user-attachments/assets/26398f42-b2cc-4dce-a786-85b50c8a6971

**Comment by h4rldev** [comment]
Yeah, the official package doesn't build with cava support for some reason

**Comment by DanteDragan** [comment]
@kernaltrap8 you could try to clear sensitivity setting with leaving "autosens": 1 or decreasing sensitivity to something around 10.

**Comment by LukashonakV** [comment]
Hi @kernaltrap8 , in order to have cava functionality is working you should enable this option before the code building for your system. According to the [Gentoo waybar-0.10.4](https://github.com/gentoo/gentoo/blob/2aa5dc44303090a50cde94e7afac9a9cda90d68f/gui-apps/waybar/waybar-0.10.4.ebuild#L81) cava functionality is disabled by default. You can use [Basic guide to write Gentoo Ebuilds](https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds) to have you own modified waybar ebuild with enabled cava... or just make and install waybar from the source.

**Comment by Alexays** [comment]
Closing — this was resolved/answered in the thread (config, environment, external dependency, or upstream — not a Waybar bug still open). Feel free to reopen if you're still stuck. 🙏
